src/js/tabs/configuration.js

```diff
@@ -453,31 +453,22 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             self.analyticsChanges['EscProtocol'] = newValue;
 
             //hide not used setting for DSHOT protocol
-            if (escProtocolValue >= self.DSHOT_PROTOCOL_MIN_VALUE) {
-                $('div.minthrottle').hide();
-                $('div.maxthrottle').hide();
-                $('div.mincommand').hide();
-                $('div.checkboxPwm').hide();
-                $('div.unsyncedpwmfreq').hide();
-
-                $('div.digitalIdlePercent').show();
+            let digitalProtocol = (escProtocolValue >= self.DSHOT_PROTOCOL_MIN_VALUE); 
+            let dshotProtocol = digitalProtocol && (escProtocolValue < self.PROSHOT_PROTOCOL_VALUE);
 
-                $('div.checkboxDshotBidir').toggle(semver.gte(CONFIG.apiVersion, "1.42.0") && escProtocolValue < self.PROSHOT_PROTOCOL_VALUE);
-                $('div.motorPoles').toggle(semver.gte(CONFIG.apiVersion, "1.42.0"));
+            $('div.minthrottle').toggle(!digitalProtocol);
+            $('div.maxthrottle').toggle(!digitalProtocol);
+            $('div.mincommand').toggle(!digitalProtocol);
+            $('div.checkboxPwm').toggle(!digitalProtocol);
+            $('div.unsyncedpwmfreq').toggle(!digitalProtocol);
 
-            } else {
-                $('div.minthrottle').show();
-                $('div.maxthrottle').show();
-                $('div.mincommand').show();
-                $('div.checkboxPwm').show();
-                //trigger change unsyncedPWMSwitch to show/hide Motor PWM freq input
-                $("input[id='unsyncedPWMSwitch']").change();
+            $('div.digitalIdlePercent').toggle(digitalProtocol);
 
-                $('div.digitalIdlePercent').hide();
+            $('div.checkboxDshotBidir').toggle(semver.gte(CONFIG.apiVersion, "1.42.0") && dshotProtocol);
```

Comment:
Not sure if this is right - bidirectional Dshot works with ProShot, but only if bitbanged Dshot is not enabled (i.e. it will have to be configured in CLI). But it should probably still show here.

Reply:
Ok, removed the proshot verification, now DShot and ProShot are the same for the bidir configuration.